frontend/src/components/Dashboard.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import DataViz from "./DataViz";
+import Expenses from "./Expenses";
+import Income from "./Income";
+
+const Dashboard = () => {
+  const containerStyles = `container mx-auto`;
```

Comment:
Should I extract the css variables to a new js module since I reuse this containerStyles variable a few times or do you mean to pull it outside of the Dashboard object? Does this make it a global variable?

Reply:
You could extract it into a new js module or just pull it out of the Dashboard object. There won't be a need to export container styles, so it's a variable isolated to this file.